doc/cephfs/add-remove-mds.rst

```diff
@@ -15,9 +15,9 @@ Provisioning Hardware for an MDS
 ================================
 
 The present version of the MDS is single-threaded and CPU-bound for most
-activities, including responding to client requests. Even so, an MDS under the
-most aggressive client loads still uses about 2 to 3 CPU cores. This is due to
-the other miscellaneous upkeep threads working in tandem.
+activities, including responding to client requests. An MDS under the most
+aggressive client loads uses about 2 to 3 CPU cores. This is due to the other
+miscellaneous upkeep threads working in tandem.
```

Comment:
Note that these comments will likely bitrot over time. CPU cores today are much faster than the ones from just a few years ago.